### PR TITLE
Add CLI for the modify endpoint

### PR DESCRIPTION
**Pull request by lubosmj**
[noissue]

### CODE REVIEW

**Comment by lubosmj** [comment]
With regards to our latest discussion, I leant towards using the `content` option instead of `modify`. Thus, a user does not have to specify the content which will be added/removed as `[{'ref': {'name': 'stable'}}, {'commit': {'checksum': '123123asfd123'}}]`, but rather she or he uses one command for adding/removing either a commit or ref (as shown in the tests).

File: tests/scripts/pulp_ostree/test_modify_content.sh [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  --ref "stable"
+
+expect_succ pulp ostree repository content remove --repository "cli_test_ostree_repository2" \
+  --commit "506f7811e94cb0966ada5f52a60eb4c34e534037497390ec4491712c3b040938"
+expect_succ pulp ostree repository content remove --repository "cli_test_ostree_repository2" \
+  --ref "stable"
+
+expect_succ pulp ostree repository destroy --name "cli_test_ostree_repository1"
+expect_succ pulp ostree repository destroy --name "cli_test_ostree_repository2"
+expect_succ pulp ostree remote destroy --name "cli_test_ostree_remote"
```
Comment (gerrod3) [inline tests/scripts/pulp_ostree/test_modify_content.sh:31]: These are not needed at the end.

**Review by gerrod3 — COMMENTED** [review]
Looks good. Are you intentionally leaving out the modify options? The command will still show up, but without any add/remove options it'll be useless.

**Comment by lubosmj** [comment]
Yes, I am intentionally leaving the modify option. Is there any simple way of disabling it even from the options list completely?

**Comment by lubosmj** [comment]
The issue with the modify option was addressed in the `pulp-cli` repository (https://github.com/pulp/pulp-cli/pull/420).

File: pulpcore/cli/ostree/repository.py [hunk 2]
```diff
@@ -81,6 +87,33 @@ def repository(ctx: click.Context, pulp_ctx: PulpContext, repo_type: str) -> Non
 repository.add_command(label_command(decorators=nested_lookup_options))
 
 
+def ref_name_callback(ctx: click.Context, param: click.Parameter, value: str) -> str:
+    if value:
+        pulp_ctx = ctx.find_object(PulpContext)
+        assert pulp_ctx is not None
+        ctx.obj = PulpOstreeRefContentContext(pulp_ctx, entity={"name": value})
```
Comment (mdellweg) [inline pulpcore/cli/ostree/repository.py:94]: I think, "name" alone is not enough to identify a ref uniquely in pulp, no?

Comment (lubosmj) [reply pulpcore/cli/ostree/repository.py:94]: Yep, you are right. The checksum of a referenced commit should be specified to make it unique. I am opening a new PR with the fix in no time.

**Pull request merged**